apps/site/src/pages/account/api-keys-panel/api-keys-list/api-keys-memoized-items.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box } from "@mui/system";
+import { Fragment, memo } from "react";
+
+import { ApiKeyItemProps, ApiKeyProps } from "../types";
+import { ApiKeyTableRow } from "./api-key-table-row";
+import { MobileApiKeyItem } from "./mobile-api-key-item";
+
+const MemoizedApiKeyItem = memo(
```

Comment:
This will be listed as 'anonymous' in the react dev tools – the function passed to memo must be named for it to have a name in dev tools. 
<img width="266" alt="image" src="https://user-images.githubusercontent.com/719814/232832163-946c5c84-f2ce-49a1-8156-44f52db225f5.png">